docs/quality_assurance.md

```diff
@@ -41,28 +41,28 @@ IBC packets:
+| 2.01 | Create IBC clients | `Scheduled` (ibc-go team) | `Done` | `??` | `Future work` | `Scheduled` | 
+| 2.02 | Getting consumer `UnbondingPeriod` from IBC client | `Scheduled` (ibc-go team) | `Done`, see TestUnbondingTime` | `??` | `NA` | `NA` | 
+| 2.03 | Create CCV channel (handshake) | `Scheduled` (ibc-go team) | `Done` | `NA` | `Future work` | `Scheduled` | 
+| 2.04 | Sending IBC packets <br /> - see `x/ccv/utils/utils.go:SendIBCPacket()` | `Scheduled` (ibc-go team) | `??` | `Done` | `Done` | `Scheduled` |
+| 2.05 | Handling acknowledgments | `Scheduled` (ibc-go team) | `Partial Coverage` | `Partial coverage` | `Scheduled` | `Scheduled` |
+| 2.06 | Handling timeouts | `Scheduled` (ibc-go team) | `??` |`??` | `Future work` | `Scheduled` |
+| 2.07 | **Handling IBC client expiration** | `Scheduled` (ibc-go team) <br /> high priority | `??` | `??` | `Future work` | `Scheduled` |
+| 2.08 | ICS-20 channel creation | `Scheduled` (ibc-go team) | `??` | `??` |`Future work` | `Scheduled` | 
+| 2.09 | ICS-20 transfer | `Scheduled` (ibc-go team) | `??` | `??` | `NA` | `Scheduled` | 
+| 2.10 | Changes in IBC-GO testing suite | `Scheduled` (ibc-go team) | `NA` | `??` | `Partial coverage` | `NA` | 
 
 ### Integration with Cosmos SDK
 
 A prerequisite of the code review is to open a PR with all the [SDK changes](https://github.com/cosmos/cosmos-sdk/tree/interchain-security-rebase) needed by Interchain Security.
 
-| ID | Concern | Code Review | Unit Testing | Diff. testing | Testnet | 
-| -- | ------- | ----------- | ------------ | ------------- | ------- | 
-| 3.01 | Changes to staking module | `Scheduled` (sdk team) | `Partial coverage` <br /> see [unbonding_test.go](../x/ccv/provider/unbonding_test.go) <br />  redelegation and validator unbonding missing | `Partial coverage` | `Scheduled` | 
-| 3.02 | Changes to slashing module | `Scheduled` (sdk team) | `Done` <br /> see [TestValidatorDowntime](../x/ccv/consumer/keeper/keeper_test.go#L345) <br />  | `NA` | `Scheduled` | 
-| 3.03 | Changes to evidence module | `Scheduled` (sdk team) | `Done` <br /> see [TestValidatorDoubleSigning](../x/ccv/consumer/keeper/keeper_test.go#L427) <br />  | `NA` | `Scheduled` | 
+| ID | Concern | Code Review | Unit Testing | E2E Testing | Diff. Testing | Testnet | 
+| -- | ------- | ----------- | ------------ | ----------- | ------------- | ------- |
+| 3.01 | Changes to staking module | `Scheduled` (sdk team) | `??` | `Partial coverage` <br /> see [unbonding_test.go](../tests/e2e/unbonding_test.go) <br /> redelegation could be expanded, validator unbonding missing | `Partial coverage` | `Scheduled` | 
+| 3.02 | Changes to slashing module | `Scheduled` (sdk team) | `??` | `Done` <br /> see [TestValidatorDowntime](../tests/e2e/slashing_test.go#L502) <br />  | `NA` | `Scheduled` | 
+| 3.03 | Changes to evidence module | `Scheduled` (sdk team) | `??` | `Done` <br /> see [TestValidatorDoubleSigning](../tests/e2e/slashing_test.go#L584) <br />  | `NA` | `Scheduled` | 
```

Comment:
Ditto: I'm a bit wary of linking/specifying test names. (Although I see this was already done before your PR so not on you!)

Reply:
Hmm yea I agree with you that linking hardcoded test names and specific lines of code is not ideal. On the other hand, the links may save people time searching through files, and I can't really think of a better alternative. Lmk if one exists 🤔